gammapy/catalog/gammacat.py

```diff
@@ -142,20 +152,25 @@ class GammaCatNotFoundError(OSError):
 class SourceCatalogGammaCat(SourceCatalog):
     """
     Gammacat open TeV sources catalog.
+
+    See: https://github.com/gammapy/gamma-cat
+
+    Parameters
+    ----------
+    filename : str
+        Path to the gamma-cat fits file.
     """
     name = 'gamma-cat'
-    description = 'an open catalog of gamma-ray sources'
+    description = 'An open catalog of gamma-ray sources'
     source_object_class = SourceCatalogObjectGammaCat
 
-    def __init__(self, filename=None):
-        if not filename:
-            if not 'GAMMA_CAT' in os.environ:
-                msg = 'The gamma-cat repo is not available. '
-                msg += 'You have to set the GAMMA_CAT environment variable '
-                msg += 'to point to the location for it to be found.'
-                raise GammapyCatFoundError(msg)
-            else:
-                filename = Path(os.environ['GAMMA_CAT']) / 'docs/data/gammacat.fits.gz'
+    def __init__(self, filename='$GAMMA_CAT/docs/data/gammacat.fits.gz'):
+        filename = make_path(filename)
+        if not filename.exists():
+            msg = 'The gamma-cat repo is not available. '
+            msg += 'You have to set the GAMMA_CAT environment variable '
+            msg += 'to point to the location for it to be found.'
+            raise GammapyCatFoundError(msg)
```

Comment:
In Python 3 it's:
```
>>> open('spam.txt')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'spam.txt'
```

So `GammaCatFoundError` -> `GammaCatNotFoundError`?

Also - I'm not sure if such a specific errors is useful. Just a generic `DataNotFoundError` in Gammapy would be OK for all catalogs, no? But I'm not sure ... put what you think best!